src/components/Link/Link.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 
 import {classNames} from '@shopify/react-utilities';
+import {ChevronLeftMinor} from '@shopify/polaris-icons';
```

Comment:
This needs to be changed to `ExternalSmallMinor` once this icon is included in Polaris Icons. I believe this is the last blocker for this PR.